vm/vm.go

```diff
@@ -168,6 +187,60 @@ type CallInfo struct {
 	Calldata        []felt.Felt
 }
 
+type FeeTokenAddresses struct {
+	EthFeeTokenAddress  felt.Felt
+	StrkFeeTokenAddress felt.Felt
+}
+
+func FeeTokenAddressesFromGateway(feederGateway *feeder.Client) (FeeTokenAddresses, error) {
+	feeTokenAddresses, err := feederGateway.FeeTokenAddresses(context.Background())
+	if err != nil {
+		return FeeTokenAddresses{}, err
+	}
+
+	return FeeTokenAddresses{
+		EthFeeTokenAddress:  feeTokenAddresses.EthL2TokenAddress,
+		StrkFeeTokenAddress: feeTokenAddresses.StrkL2TokenAddress,
+	}, nil
+}
+
+func DefaultFeeTokenAddresses() FeeTokenAddresses {
```

Comment:
Can we convert this to var? If yes, I think we can also replace the [4]uint64 by a function call to the hex numbers, as it's only computed once.